### DISCUSSION TITLE
Is it possible to get legal data?

### DISCUSSION THREAD

**Discussion by jimenasalinas** [ip-tools/python-epo-ops-client#111] [Q&A] [answered]
Hi, thank you for this library. Is it possible to get data from the legal EPO APIs?

**Comment by amotl** [comment, marked as answer]
Hi Jimena,

thanks to a patch by @MartiONE,

- https://github.com/ip-tools/python-epo-ops-client/pull/133

the improvement has been released with python-epo-ops-client 4.2.0.

With kind regards,
Andreas.
